### PR TITLE
Add status codes from neo4j PR 28647

### PR DESCRIPTION
**Pull request by sherfert**

### CODE REVIEW

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 1]
```diff
@@ -1403,6 +1403,18 @@ Status description:: error: system configuration or operation exception - bolt i
 
 Status description:: error: system configuration or operation exception - unsupported operation of a sharded database. Feature: `{ $feat }` is not available in a sharded database.
 
+=== 51N72
+
+Status description:: error: system configuration or operation exception - memory pool out of memory. Failed to allocate memory in a memory pool. See `{ $cfgSetting }` in the neo4j configuration.
+
+=== 51N73
+
+Status description:: error: system configuration or operation exception - transaction memory limit reached. The transaction used more memory than was allowed. The maximum allowed size for a transaction can be configured with `{ $cfgSetting }` in the neo4j configuration.
```
Comment (renetapopova) [inline modules/ROOT/pages/errors/gql-errors.adoc:1412]:
```suggestion
Status description:: error: system configuration or operation exception - transaction memory limit reached. The transaction uses more memory than it is allowed. The maximum allowed size for a transaction can be configured with `{ $cfgSetting }` in the neo4j.conf file.
```

Comment (renetapopova) [reply modules/ROOT/pages/errors/gql-errors.adoc:1412]: We try to use present tense whenever possible. I don't know if this message has already been merged in the monorepo.

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 2]
```diff
@@ -1403,6 +1403,18 @@ Status description:: error: system configuration or operation exception - bolt i
...
+
+=== 51N74
+
+Status description:: error: system configuration or operation exception -maximum number of transactions reached. Failed to start a new transaction. The limit of concurrent transactions is reached. Increase the number of concurrent transactions using `{ $cfgSetting }` in the neo4j configuration.
```
Comment (renetapopova) [inline modules/ROOT/pages/errors/gql-errors.adoc:1416]:
```suggestion
Status description:: error: system configuration or operation exception - maximum number of transactions reached. Failed to start a new transaction. The limit of concurrent transactions is reached. Increase the number of concurrent transactions using `{ $cfgSetting }` in the neo4j.conf file.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-240.surge.sh

Updated pages:
- [List of GQLSTATUS error codes](https://neo4j-docs-status-codes-240.surge.sh/status-codes/2025.01/errors/gql-errors)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-240.surge.sh

Updated pages:
- [List of GQLSTATUS error codes](https://neo4j-docs-status-codes-240.surge.sh/status-codes/2025.01/errors/gql-errors)

**Review by renetapopova — APPROVED** [review]
Looks good.

**Pull request merged**